1.6/Source/PawnStorages/PawnStorages/CompPawnStorageNutrition.cs

```diff
@@ -269,33 +399,52 @@ public override IEnumerable<Gizmo> CompGetGizmosExtra()
                 nutritionComp = this,
                 defaultLabel = "PS_CommandSetNutritionLevel".Translate(),
                 defaultDesc = "PS_CommandSetNutritionLevelDesc".Translate(),
-                icon = CompRefuelable.SetTargetFuelLevelCommand
+                icon = CompRefuelable.SetTargetFuelLevelCommand,
             };
         }
-        if (!DebugSettings.ShowDevGizmos) yield break;
+        if (!DebugSettings.ShowDevGizmos)
+            yield break;
         yield return new Command_Action
         {
-            defaultLabel = "Feed Now", action = DoFeed, icon = ContentFinder<Texture2D>.Get("UI/Buttons/ReleaseAll")
+            defaultLabel = "Feed Now",
```

Comment:
I'm wondering if it might be worth letting this one exist outside of dev mode. Feels like giving users a way to force these things might get a few requests gone. Maybe this and the absorb now one. Refactor a bunch into a menu because these things end up with a loooot of buttons.